api/src/pcapi/routes/backoffice/products/blueprint.py

```diff
@@ -204,6 +205,7 @@ def get_product_details(product_id: int) -> utils.BackofficeResponse:
         rejected_offers_count=rejected_offers_count,
         ineligibility_reasons=ineligibility_reasons,
         product_whitelist=product_whitelist,
+        music_titelive_subcategory=subcategories.MUSIC_TITELIVE_SUBCATEGORY_SEARCH_IDS,
```

Comment:
Je préfère avoir une variable au pluriel pour bien comprendre qu'elle contient une collection, et non une sous-catégorie : 
```suggestion
        music_titelive_subcategories=subcategories.MUSIC_TITELIVE_SUBCATEGORY_SEARCH_IDS,
```